### PR TITLE
Gildas: update to 201905a

### PR DESCRIPTION
**Pull request by bardeau**
@smaret can you please merge this PR for Gildas may19 release? Thanks.

### CODE REVIEW

File: science/gildas/Portfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ PortGroup           compilers 1.0
 PortGroup           active_variants 1.1 
 
 name                gildas
-version             201904a
+version             201905a
 revision            1
```
Comment (mf2k) [inline science/gildas/Portfile]: The `revision` needs to be reset to 0.

Comment (bardeau) [reply science/gildas/Portfile]: I have removed the 'revision' in the following commit. Is this enough to accept the PR?

Comment (mf2k) [reply science/gildas/Portfile]: Yes, now you just need to squash the commits into 1.

Comment (bardeau) [reply science/gildas/Portfile]: Can I do this with the GitHub interface?

Comment (markemer) [reply science/gildas/Portfile]: Run `git rebase -i HEAD~2`, then change one of the `pick`s to `squash` and save the open file. Fix the commit message that comes up, and then `git push --force`

Comment (bardeau) [reply science/gildas/Portfile]: Ok thanks. I was just wondering what happens in GitHub when I overwrite the branch (git push --force). This is done.

**Comment by mf2k** [comment]
Thank you!

**Pull request merged**